src/scm/Model.ts

```diff
@@ -341,19 +341,22 @@ export class Model implements Disposable {
     }
 
     private async getFstatInfoForFiles(files: string[]): Promise<any> {
-        const fstatOutput: string = await Utils.getOutput(`fstat ${files.join(' ')}`);
-        // Windows will have lines end with \n\r.
+        const fstatOutput: string = await Utils.getOutput(`fstat "${files.join('" "')}"`);
+        // Windows will have lines end with \r\n.
         // Each file has multiple lines of output separated by a blank line.
-        // Splitting on \n\r?\n will separate the output for each file.
+        // Splitting on \n\r?\n will find newlines followed immediately by a newline
+        // which will split the output for each file.
         const fstatFiles = fstatOutput.trim().split(/\n\r?\n/);
         return fstatFiles.map((file) => {
             const lines = file.split('\n');
             const lineMap = {};
             lines.forEach(line => {
-                // ... Name Value
-                const matches = line.match(/[.]{3} (\w+) (.+)/);
+                // ... Key Value
+                const matches = line.match(/[.]{3} (\w+) (.+)?/);
```

Comment:
The space shouldn't be captured as part of the capture group, though, so I'll use this: `[/[.]{3} (\w+)[ ]*(.+)?/`

Does that look ok?